### PR TITLE
Add ability to customize health check path for target groups.

### PR DESCRIPTION
**Pull request by owms**
A better way to do this in the future might be to require json definition files for the target group to be created similar to the json definition files expected for the task and service definitions.  Right now however we only need to update 1 param on the target group (health check path) and requiring a new json definition file for all our services would be a bit of work to create those.

### CODE REVIEW

File: ecsUpdate.py [hunk 1]
```diff
@@ -60,15 +60,15 @@ def load(self,cluster, region, name, env, job_path, handle_file):
...
+                            handle_file(client, cluster, ecs_json, env, region, filename, count, elb_client, healthcheckpath)
                         except:
                             logging.exception("Error reading file %s " % filename)
                             self.catfile(filename)
                             raise
 
 
     @staticmethod
-    def register_task(client, cluster, ecs_json, env, region, filename, count=0, elb_client=None):
+    def register_task(client, cluster, ecs_json, env, region, filename, count, elb_client, healthcheckpath):
```
Comment (owap) [inline ecsUpdate.py]: Did we mean to remove the default values for `count` and `elb_client`?

Comment (owms) [inline ecsUpdate.py]: Yes, python was complaining about adding a new parameter without a default value after parameters that have default values.

Comment (owap) [inline ecsUpdate.py]: Oh so we could just default `healthcheckpath` to `'/'` or something then

Comment (owap) [inline ecsUpdate.py]: @owms mentioned to me in Slack that those params aren't even used, they just exist to maintain the same signature as `register_service`, so LGTM

**Pull request merged**